make/modules/java.base/gendata/GendataTZDB.gmk

```diff
@@ -30,7 +30,7 @@ GENDATA_TZDB :=
 #
 TZDATA_DIR := $(MODULE_SRC)/share/data/tzdata
 TZDATA_TZFILE := africa antarctica asia australasia europe northamerica southamerica backward etcetera gmt jdk11_backward
-TZDATA_TZFILES := $(addprefix $(TZDATA_DIR)/,$(TZDATA_TZFILE))
+TZDATA_TZFILES := $(TZDATA_DIR)/*
```

Comment:
Have you verified that this works? I would expect a wildcard call to be needed. 
```suggestion
TZDATA_TZFILES := $(wildcard $(TZDATA_DIR)/*)
```

Reply:
Hm, yes, I think there are some cases where wildcards will be resolved without a call to wildcard. I'm also not sure that kind of expansion works the same across make versions, and if you ever needed to use that file list for anything else, it would be easy to trip up. By being explicit with the call, we know what we are getting.